frontend/public/components/volumes-table.tsx

```diff
@@ -209,6 +144,78 @@ export const VolumesTable = (props) => {
       props: { className: volumeRowColumnClasses[6] },
     },
   ];
+  const VolumesTableRows = ({ componentProps: { componentData } }) => {
+    return _.map(componentData, (volume: RowVolumeData) => {
+      const {
+        container,
+        mountPath,
+        name,
+        readOnly,
+        volumeResource,
```

Comment:
I realize you're trying to avoid shadowing variables from the outer scope, but `volumeResource` and `volumePod` are confusing names. It sounds like they refer to a `PersistentVolume` or container storage. I would pick something different.

Reply:
All set.